app/models/async_transaction/base.rb

```diff
@@ -13,6 +15,8 @@ class Base < ApplicationRecord
     }
 
     attr_encrypted :metadata, key: Settings.db_encryption_key
+    serialize :metadata, JsonMarshal::Marshaller
```

Comment:
Do we need to add the `serialize` lines while we're migrating?

Reply:
Yes because if new records are created during the migration process, lockbox will properly save the encrypted data into the lockbox field (as well as in the attr_encrypted field).